### DISCUSSION TITLE
blog/Vue%EC%99%80%20React%EC%9D%98%20%EC%83%81%ED%83%9C%EA%B4%80%EB%A6%AC

### DISCUSSION THREAD

**Discussion by giscus** [khj0426/HJ_Devlog#9] [General]
# blog/Vue%EC%99%80%20React%EC%9D%98%20%EC%83%81%ED%83%9C%EA%B4%80%EB%A6%AC

어떻게 상태가 다를까!!

https://hj-devlog.vercel.app/blog/Vue%EC%99%80%20React%EC%9D%98%20%EC%83%81%ED%83%9C%EA%B4%80%EB%A6%AC

**Comment by oaoong** [comment]
오 이런 중요한 차이가 있었군요. react에서 레퍼런스 타입의 prop을 저런식으로 처리하는지도 몰랐는데 덕분에 알아갑니다!
